### PR TITLE
Reduce package size by 1MB

### PR DESCRIPTION
**Pull request by timneutkens**
Moved all dx modules to micro-dev

old:
old

new:
new

### CODE REVIEW

**Comment by floatdrop** [comment]
@timneutkens can you describe, how developer should now use `micro`? Also, why `args` is not dx module too?

File: lib/index.js [hunk 1]
```diff
@@ -9,33 +6,28 @@ const getModule = require('./module');
 module.exports = async (file, flags, module = getModule(file)) => {
```
Comment (albinekb) [inline lib/index.js:6]: this does not need to be `async` anymore since there's no `await` in it

**Comment by floatdrop** [comment]
As side note – we can reduce package even more splitting `args`, `is-async-supported`, `node-version` and  `async-to-gen` to `micro-cli`.

**Comment by timneutkens** [comment]
See #260

**Pull request closed without merging**